releasenotes/README.md

```diff
@@ -5,6 +5,23 @@ Notes should be created as part of the pull request for any user facing changes.
 a release, the release notes utility will be run in order to generate a release notes file
 which will be reviewed by the release managers and documentation team.
 
+
+## When to Add Release Notes
+
+Release notes should be added for any user facing changes. These include:
```

Comment:
Kubernetes has similar phrasing here: https://www.kubernetes.dev/docs/guide/release-notes/ and I thought that it summarizes what we add release notes quite well.